cmd/gapit/trace.go

```diff
@@ -237,7 +236,6 @@ func (verb *traceVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 		ObserveDrawFrequency:         uint32(verb.Observe.Draws),
 		StartFrame:                   uint32(verb.Start.At.Frame),
 		FramesToCapture:              uint32(verb.Capture.Frames),
-		DisablePcs:                   verb.Disable.PCS,
 		RecordErrorState:             verb.Record.Errors,
```

Comment:
This is dead now too, right?

Reply:
Good catch, created b/155718960 to track this and all GL error related leftovers.